src/components/migrationToV3.stories.mdx

```diff
@@ -437,6 +442,140 @@ When you implement your own theme, you need to use `globalStyle` instead of `glo
+| `MeetingManager.devicePermissionsObservers`                 | `MeetingManager.deviceLabelTriggerStatusObservers`         |
+| `MeetingManager.subscribeToDevicePermissionStatus()`        | `MeetingManager.subscribeToDeviceLabelTriggerStatus()`     |
+| `MeetingManager.unsubscribeFromDevicePermissionStatus()`    | `MeetingManager.unsubscribeFromDeviceLabelTriggerStatus()` |
+|                                                             |                                                            |
+| `MeetingManager.deviceLabelTriggerChangeObservers`          | `MeetingManager.deviceLabelTriggerObservers`               |
+| `MeetingManageer.subscribeToDeviceLabelTriggerChange()`     | `MeetingManageer.subscribeToDeviceLabelTrigger()`          |
+| `MeetingManageer.unsubscribeFromDeviceLabelTriggerChange()` | `MeetingManageer.unsubscribeFromDeviceLabelTrigger()`      |
+
+### Update Type of Selected Device in MeetingManager
+
+In V3, we improve type for selected device in `MeetingManager` and update the corresponding components and hooks.
+
+#### MeetingManager Properties Change
+
+| Before in 2.x                              | After in 3.x                                             |
+| ------------------------------------------ | -------------------------------------------------------- |
+| `selectedAudioOutputDevice: string | null` | `selectedAudioOutputDevice: string | null`               |
+| `selectedAudioInputDevice: string | null`  | `selectedAudioInputDevice: AudioInputDevice | undefined` |
+| `selectedVideoInputDevice: string | null`  | `selectedVideoInputDevice: VideoInputDevice | undefined` |
+
+```js
+type Device = string | MediaTrackConstraints | MediaStream;
+type AudioInputDevice = Device | AudioTransformDevice | null;
+type VideoInputDevice = Device | VideoTransformDevice;
+```
+#### Hooks Return Values Change
+
+Before in 2.x
+
+```js
+// Selected audio output deivce. The type is `string | null`
+const { selectedDevice } = useAudioOutputs();
+// Selected audio input deivce. The type is `string | null`
+const { selectedDevice } = useAudioInputs();
+// Selected video input deivce. The type is `string | null`
+const { selectedDevice } = useVideoInputs();
+```
+
+After in 3.x
+
+```js
+// Selected audio output deivce. The type is `string | null`
+const { selectedDevice } = useAudioOutputs();
+// Selected audio input deivce. The type is `AudioInputDevice | undefined`
+const { selectedDevice } = useAudioInputs();
+// Selected video input deivce. The type is `VideoInputDevice | undefined`
+const { selectedDevice } = useVideoInputs();
+```
+
+### Remove `useSelectAudioInputDevice`, `useSelectAudioOutputDevice` and `useSelectVideoInputDevice` hook
+
+In V3, we remove `useSelectAudioInputDevice`, `useSelectAudioOutputDevice` and `useSelectVideoInputDevice` hook. These hooks just expose device selection APIs from `MeetingManager`, thus they are redundant.
```

Comment:
nit: Since we have done it suggest using past tense:
In V3, we have removed `useSelectAudioInputDevice`, `useSelectAudioOutputDevice` and `useSelectVideoInputDevice` hook. These hooks just exposed device selection APIs from `MeetingManager`, thus they were redundant.

Reply:
Updated all tenses.